rflx/cli.py

```diff
@@ -165,3 +172,9 @@ def graph(args: argparse.Namespace) -> None:
     filename = Path(directory).joinpath("locations.json")
     with open(filename, "w") as f:
         json.dump(locations, f)
+
+
+def export(args: argparse.Namespace) -> None:
+    model = parse(args.files)
+    with open(args.output, "w") as f:
+        print(json.dumps(model.serialize, indent=3), file=f)
```

Comment:
Is there a reason for using `print` instead of `json.dump(model.serialize, f, indent=3)`?

Reply:
No, changed in 721c9ed.